src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -760,12 +763,14 @@ private class ACCT_IndividualAccounts_TEST {
         insert con;
 
         Test.startTest();
-        con.AccountId = null;
+        con.AccountId = NULL;
         update con;
         Test.stopTest();
 
-        Account assertAccount = [Select Primary_Contact__c FROM Account LIMIT 1];
-        system.assertEquals(null, assertAccount.Primary_Contact__c);
+        Account assertAccount = [SELECT Primary_Contact__c FROM Account LIMIT 1];
+        Contact[] relatedCOntact = [SELECT AccountId FROM Contact];
```

Comment:
Minor -- I believe we prefer <List>'s over [] notation. Consider updating all references in this class to use Lists